modules/installation/pages/ports.adoc

```diff
@@ -1,5 +1,5 @@
 [[ports]]
-= Required Network Ports
+= Network Ports
```

Comment:
I think we must add port `9090`.

Reply:
I'd keep it as it is or mark as _optional_. We normally don't deploy Prometheus server on the client. Federation is a special case which possibly will be covered when working on cluster awareness story.

Reply:
> I think we must add port `9090`.

What's on port 9090?

Reply:
> I'd keep it as it is or mark as _optional_. We normally don't deploy Prometheus server on the client. Federation is a special case which possibly will be covered when working on cluster awareness story.

This is the heading for the entire section. I realised when discussing it in the PHP that this whole page lists ports that are, for the most part, optional. We even have a note stating that at the top: 

> You will not need to open all of these ports.
> Some ports only need to be opened if you are using the service that requires them.

So I removed "Required" because, basically, they aren't.

Reply:
Port 9090 is where the Prometheus UI runs - it is exposed by the Prometheus server itself.